### PR TITLE
47fix

### PR DESCRIPTION
**Pull request by yapei**

### CODE REVIEW

File: features/step_definitions/admin_console.rb [hunk 1]
```diff
@@ -27,6 +27,7 @@
                                password: user.password,
                                idp: env.idp)
   raise "cannot login to cluster console" unless @result[:success]
+  step "I run the :navigate_to_admin_console web action"
```
Comment (yapei) [inline features/step_definitions/admin_console.rb:30]: Added step to switch to admin console for every user @yanpzhan can you see any potential issue for this update?

File: lib/rules/web/admin_console/4.5/goto.xyaml [hunk 2]
```diff
@@ -241,6 +241,7 @@ goto_installed_operators_page:
   action: wait_box_loaded
 goto_all_installed_operators_page:
   url: k8s/all-namespaces/operators.coreos.com~v1alpha1~ClusterServiceVersion
+  action: wait_table_loaded
```
Comment (yapei) [inline lib/rules/web/admin_console/4.5/goto.xyaml:244]: wait until resource table is loaded before going next step

File: lib/rules/web/admin_console/4.7/goto.xyaml [hunk 3]
```diff
@@ -261,6 +265,7 @@ goto_installed_operators_page:
   action: wait_box_loaded
 goto_all_installed_operators_page:
   url: k8s/all-namespaces/operators.coreos.com~v1alpha1~ClusterServiceVersion
+  action: wait_table_loaded
```
Comment (yapei) [inline lib/rules/web/admin_console/4.7/goto.xyaml:268]: OCP-29724 affected

File: lib/rules/web/admin_console/4.7/monitoring.xyaml [hunk 4]
```diff
@@ -8,7 +8,7 @@ send_metrics_query:
     op: send_keys <metrics_name>
 query_metrics:
   params:
-    button_text: Run Queries
+    button_text: Run queries
```
Comment (yapei) [inline lib/rules/web/admin_console/4.7/monitoring.xyaml:11]: OCP-25816 is affected

File: features/step_definitions/admin_console.rb (hunk 1 above)
Comment (yanpzhan) [reply features/step_definitions/admin_console.rb:30]: No harm from my view for now.

**Review by pruan-rht — COMMENTED** [review]
/lgtm

**Pull request merged**